lib/synapse/service_watcher.rb

```diff
@@ -1,13 +1,17 @@
 require "synapse/log"
-require "synapse/service_watcher/base"
+require "synapse/service_watcher/base/base"
```

Comment:
NIT: for base one, just leave it under `synapse/service_watcher/base`? other derived watchers should have own folder

Reply:
I avoided doing that because the `service_watcher.rb` loader requires a common path structure for every type, including the base watcher.